planner/core/find_best_task.go

```diff
@@ -781,17 +781,19 @@ func (ds *DataSource) convertToIndexMergeScan(prop *property.PhysicalProperty, c
 	}
 	for _, partPath := range path.PartialIndexPaths {
 		var scan PhysicalPlan
-		var partialCost, rowCount float64
+		var partialCost float64
 		if partPath.IsTablePath() {
-			scan, partialCost, rowCount = ds.convertToPartialTableScan(prop, partPath)
+			scan, partialCost, _ = ds.convertToPartialTableScan(prop, partPath)
 		} else {
-			scan, partialCost, rowCount = ds.convertToPartialIndexScan(prop, partPath)
+			scan, partialCost, _ = ds.convertToPartialIndexScan(prop, partPath)
```

Comment:
We can simply remove the last returned result of these 2 functions?

Reply:
Because these results are only used to calculate the total row count. In this PR we directly use the calculated `CountAfterAccess` as the total row count so we don't need the results in this situation.

Reply:
Do you mean modify these two functions to only return two values?

Reply:
Fixed that. Thanks.